release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -97,9 +96,52 @@ type of private DNS zone.
 
 // For more information on this new command and migrating Azure private DNS
 // zones, see ../installing/installing_azure/migrating-azure-dns-zones.adoc#migrating-azure-dns-zones[Migrating legacy Azure DNS zones].
-////
-// Monitor whether this info moves to a KB article or remains in official docs.
-// If it moves to KB, permanently remove this entry.
+
+[id="ocp-4-5-built-in-help-for-installconfig-supported-fields"]
+==== Built-in help for InstallConfig supported fields
```

Comment:
Is "InstallConfig" actually a thing? I don't see any references in our docs on it. Is this talking about the install-config.yaml file?

Reply:
Good point; I'll phrase it differently since that doesn't seem to be the standard way :+1: